docs/docs/styled-components.md

```diff
@@ -114,7 +114,7 @@ export default () => (
 
 ### Enabling user stylesheets with a stable class name
 
-Adding a persistent CSS `className` to your styled components can make it easier for users to take advantage of [User Stylesheets](https://www.viget.com/articles/inline-styles-user-style-sheets-and-accessibility/) for accessibility.
```

Comment:
I personally don't think that "user stylesheets" should be capitalized, but I'm not super committed either way.